### DISCUSSION TITLE
ummmmmmmmmmmm

### DISCUSSION THREAD

**Discussion by iop3** [is-a-dev/register#3153] [Discussion] [answered]
it seems likee the maintainers are a bit inactive rn?

can i be one >:3

**Comment by omsenjalia** [comment]
> it seems likee the maintainers are a bit inactive rn?
> 
> can i be one >:3

Nope. There are a few stages if becoming maintiner's (this is not confirmed it's my theory/speculation i don't think it's true)

First rank you have to earn is Helper. After that comes Mod or lead Helper. And after that comes Admin. After that if there is a post open for Maintiner and maintiner's think they need a team member they will recruit one.

As a helper i know how hard it to to become a helpers and i was so close to becoming a maintiner. But it got lost away cuz of a better worthy guy. It's alright but as you know the maintiner's have a life that's why they can be inactive. Maybe when a new post opens for Maintiner and your a helper+ you will have a tiny chance of becoming a maintiner.

That's the basic story of how to become a maintiner. I am not sure if my info is accurate. So yea.

**Reply by iop3** [reply]
mhmn

**Reply by MaskDuck** [reply]
well, actually, we pick people we trust to become maintainer when we need one, no need to become helper/mod in discord server, just need to be trusted by the maintainers :)

**Comment by MaskDuck** [comment]
no

**Reply by iop3** [reply]
i already know tf

**Comment by mtgsquad** [comment, marked as answer]
@smileybox Depends on your activity, contributions, and how many people you help. We have a discord server if you would like to learn more. https://discord.gg/ZXB4zuRXkC

**Reply by wdhdev** [reply]
@mtgsquad I've been pretty helpful, helping people out on PR's.

**Reply by MaskDuck** [reply]
> @mtgsquad I've been pretty helpful, helping people out on PR's.

unfortunately we don't need more as of rn
